packages/@dcl/inspector/src/lib/babylon/decentraland/camera.ts

```diff
@@ -135,8 +135,11 @@ export class CameraManager {
     const center = new BABYLON.Vector3(PARCEL_SIZE / 2, 0, PARCEL_SIZE / 2)
     const size = center.length()
     const camera = new BABYLON.FreeCamera('editorCamera', center.subtractFromFloats(size, -size * 1.5, size * 2), scene)
+    const mouseInput = camera.inputs.attached.mouse as BABYLON.FreeCameraMouseInput
     camera.target = center
 
+    mouseInput.buttons = [2] // disable all buttons except right mouse button
```

Comment:
could we define this number as const?

```suggestion
    const RIGHT_BUTTON = 2;
    ...
    mouseInput.buttons = [RIGHT_BUTTON] // disable all buttons except right mouse button
```